package.json

```diff
@@ -0,0 +1,83 @@
+{
+  "name": "@eslint/json",
+  "version": "0.0.1",
+  "description": "JSON linting plugin for ESLint",
+  "author": "Nicholas C. Zakas",
+  "type": "module",
+  "main": "dist/esm/index.js",
+  "types": "dist/esm/index.d.ts",
+  "exports": {
+    "require": {
+      "types": "./dist/cjs/index.d.cts",
+      "default": "./dist/cjs/index.cjs"
+    },
+    "import": {
+      "types": "./dist/esm/index.d.ts",
+      "default": "./dist/esm/index.js"
+    }
+  },
+  "files": [
+    "dist"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "gitHooks": {
+    "pre-commit": "lint-staged"
+  },
+  "lint-staged": {
+    "*.js": [
+      "eslint --fix",
+      "prettier --write"
+    ],
+    "!(*.js)": "prettier --write --ignore-unknown"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/eslint/json.git"
+  },
+  "bugs": {
+    "url": "https://github.com/eslint/json/issues"
+  },
+  "homepage": "https://github.com/eslint/json#readme",
+  "scripts": {
+    "build:dedupe-types": "node tools/dedupe-types.js dist/cjs/index.cjs dist/esm/index.js",
+    "build:cts": "node -e \"fs.copyFileSync('dist/esm/index.d.ts', 'dist/cjs/index.d.cts')\"",
+    "build": "rollup -c && npm run build:dedupe-types && tsc -p tsconfig.esm.json && npm run build:cts",
+    "build:docs": "eslint-doc-generator --check",
+    "test:jsr": "npx jsr@latest publish --dry-run",
+    "pretest": "npm run build",
+    "lint": "eslint",
+    "test": "mocha tests/**/*.js",
+    "test:coverage": "c8 npm test"
+  },
+  "keywords": [
+    "eslint",
+    "eslint-plugin",
+    "eslintplugin",
+    "json",
+    "linting"
+  ],
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "@eslint/core": "^0.1.0",
+    "@types/eslint": "^8.56.10",
+    "c8": "^9.1.0",
+    "eslint": "github:eslint/eslint",
```

Comment:
Is this intentional?

Reply:
Oops no. Had to do that before the release.